files/etc_nagios-plugins_orig/config/check_nrpe.cfg

```diff
@@ -8,4 +8,11 @@ define command {
 define command {
 	command_name    check_nrpe_1arg
 	command_line    /usr/lib/nagios/plugins/check_nrpe -H $HOSTADDRESS$ -c $ARG1$
-}
\ No newline at end of file
+}
+
+# this command runs a program $ARG1$ with arguments $ARG2$ and $ARG3$
+define command {
+	command_name    check_nrpe_3arg
```

Comment:
I will merge, but this is unnecesary. as with only 1arg you can send many args...

like in:
´´´
define service {
    command    check_nrpe_1arg!-p1 value -p2 value -p3 value
}
so in 1arg you send many command parameters.